src/hooks/useMedia.ts

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useState } from "react";
+
+type UseMediaType = {
```

Comment:
Should be named `MediaDevices`

Reply:
Should I use `MediaDevicesType` or `MediaDevices`?

Reply:
MediaDevices should be good